### PR TITLE
linked time: Wire up timeSelectionChange action

### PR DESCRIPTION
**Pull request by JamesHollyer**
In order to allow the LinkedTimeFobController to implement dragging it must be able to dispatch the timeSelectionChanged action. This pulls that action into the ScalarCardContainer and passes it down to give the controller access.

### CODE REVIEW

File: tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts [hunk 1]
```diff
@@ -517,4 +520,13 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
       };
     });
   }
+
+  onSelectTimeChanged(linkedTime: LinkedTime) {
```
Comment (japie1235813) [inline tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts:524]: nit: consider naming the input to "newLinkedTime" or "newValue" to make it more meaningful

see 
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/views/right_pane/settings_view_container.ts;l=195?q=%22onSelectTimeChanged%22&ss=tensorflow%2Ftensorboard

**Review by japie1235813 — COMMENTED** [review]
Any tests for this change?

File: tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts (hunk 1 above)
Comment (JamesHollyer) [reply tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts]: Yea that makes sense I renamed it newLinkedTime. This should probably be done in the histogram_card_container as well but, out of scope of this PR.

**Review by JamesHollyer — COMMENTED** [review]
The only meaningful tests I can think of will come in the integration tests once this is actually being called.

**Review by japie1235813 — APPROVED** [review]
so if there is nothing meaningful to call the output onSelectTimeChanged in scalar_card for this PR, please consider putting down something meaningful so then it is testable.
"onSelectTimeChanged" dose trigger an action, so I would like to see where you test the action is triggered.

**Review by JamesHollyer — COMMENTED** [review]
The meaningful tests will be coming soon in a future PR.

**Pull request merged**